config/dev.js

```diff
@@ -1,13 +1,15 @@
 // Config for the -dev server.
 import { addonsServerDevCDN, analyticsHost, apiDevHost, baseUrlDev, devDomain } from './lib/shared';
 
-const addonsFrontendCDN = 'https://addons-amo-dev-cdn.allizom.org';
+const staticPath = '/static-frontend/';
 
 module.exports = {
   baseURL: baseUrlDev,
   apiHost: apiDevHost,
   amoCDN: addonsServerDevCDN,
-  staticHost: addonsFrontendCDN,
+  // addons-frontend statics are served by the CDN from the main domain
+  staticHost: undefined,
```

Comment:
Alternatively I could set it to `baseUrlDev`, but that seems wasteful. The only thing it would help with is local testing of the `NODE_CONFIG_ENV=dev`, but I'm not sure it's super useful.

Reply:
`staticHost` also adds a preconnect link when defined.

Reply:
> `staticHost` also adds a preconnect link when defined

Yeah. Obviously this is not necessary here since we'll already be on the same domain (there is a check that it's different from `baseURL` so it's all the same)

> we should make sure that the `STATIC_HOST` env var isn't defined in the Puppet config

It's not set in `cloudops-deployment` at least